cluster/docker-in-docker/docker-compose.yml

```diff
@@ -0,0 +1,128 @@
+ambassador:
+  image: cpuguy83/docker-grand-ambassador:0.9.1
+  volumes:
+  - /var/run/docker.sock:/var/run/docker.sock
+  command: "-name dind_apiserver_1"
+etcd:
+  hostname: etcd
+  image: quay.io/coreos/etcd:v2.2.5
+  ports: [ "4001:4001" ]
+  command: >
+    --listen-client-urls 'http://etcd:4001'
+    --advertise-client-urls 'http://etcd:4001'
+    --initial-cluster-state new
+node:
+  hostname: node
+  privileged: true
+  image: k8s.io/kubernetes-dind
+  ports: [ "10248","10249" ]
+  entrypoint:
+  - bash
+  - -xc
+  - >
+    NAME=$$(cut -f2 -d/ <<<$${APISERVER_NAME}) &&
+    N=$${NAME##*_} &&
+    HOSTNAME_OVERRIDE=$$(getent hosts node | cut -d' ' -f1 | sort -u | tail -1) &&
+    DOCKER_NETWORK_OFFSET=0.0.$${N}.0
+    exec wrapdocker bash -c "
+    hyperkube kubelet
+    --hostname-override=$${HOSTNAME_OVERRIDE}
+    --node-labels="rack=$${N},gen=201$${N}"
+    --api-servers=http://apiserver:8888
+    --allow-privileged
+    --cluster-dns=10.10.10.10
+    --cluster-domain=cluster.local
+    --config=/var/lib/kubelet/manifests &
+    hyperkube proxy
+    --hostname-override=$${HOSTNAME_OVERRIDE}
+    --master=http://apiserver:8888
+    --conntrack-max=0
+    --v=4 &
+    wait \$$(jobs -p)
+    "
+  command: []
+  environment:
+  - VAR_LIB_DOCKER_SIZE=10
+  - DOCKER_DAEMON_ARGS
+  links:
+  - etcd
+  - "ambassador:apiserver"
+apiserver:
+  hostname: apiserver
+  image: k8s.io/kubernetes-dind
+  entrypoint:
+  - /bin/bash
+  - "-ceu"
+  - >
+    echo "Hostname: $$(hostname -f) ($$(hostname -f | xargs resolveip.sh))" &&
+    await-health-check "-t=${DOCKER_IN_DOCKER_ETCD_TIMEOUT}" http://etcd:4001/health &&
+    await-file "-t=${KUBE_KEYGEN_TIMEOUT}" /var/run/kubernetes/auth/apiserver.crt &&
+    hyperkube apiserver
+    --insecure-bind-address=$$(resolveip.sh apiserver)
+    --external-hostname=apiserver
+    --etcd-servers=http://etcd:4001
+    --insecure-port=8888
+    --admission-control=NamespaceLifecycle,LimitRanger,SecurityContextDeny,ServiceAccount,ResourceQuota
```

Comment:
I don't think it was depricated. To point of removing the admission was to keep production and testing environment as close as possible. E.g. https://github.com/kubernetes/kubernetes/pull/21004